assembl/static2/js/app/pages/debate.jsx

```diff
@@ -1,12 +1,27 @@
-import React from 'react';
-import { PropTypes } from 'prop-types';
+// @flow
+import * as React from 'react';
 import { connect } from 'react-redux';
 import { graphql } from 'react-apollo';
 import Loader from '../components/common/loader';
 import Themes from '../components/debate/common/themes';
 import DebateThematicsQuery from '../graphql/DebateThematicsQuery.graphql';
 
-const Debate = ({ identifier, data, params, children }) => {
+type Props = {
+  identifier: string,
+  data: {
+    loading: boolean,
+    error: ?Error,
+    thematics: DebateThematicsQueryQuery
```

Comment:
QueryQuery ?

Reply:
it's a generated name: see the flow/graphql_types.flow.js